parent/pom.xml

```diff
@@ -669,6 +669,58 @@
       </build>
     </profile>
 
+		<profile>
+			<id>deploy-to-jboss.org</id>
+			<build>
+				<plugins>
+					<plugin>
+						<artifactId>maven-dependency-plugin</artifactId>
+						<executions>
+							<execution>
+								<goals>
+									<goal>unpack</goal>
+								</goals>
+								<phase>deploy</phase>
+								<configuration>	
+									<artifactItems>
+										<artifactItem>
+					          					<groupId>org.jboss.tools.releng</groupId>		
+											<artifactId>jbosstools-releng-publish</artifactId>
+											<version>4.3.0.Alpha2-SNAPSHOT</version>
+											<type>tar.gz</type>
+											<outputDirectory>${project.build.directory}/releng-scripts</outputDirectory>	
+										</artifactItem>
+									</artifactItems>
+								</configuration>
+							</execution>
+						</executions>
+					</plugin>
+					<plugin>
+						<groupId>org.codehaus.mojo</groupId>
+						<artifactId>exec-maven-plugin</artifactId>
+						<version>1.3.2</version>
+						<executions>
+							<execution>
+								<goals>
+									<goal>exec</goal>
+								</goals>
+								<phase>deploy</phase>
+								<configuration>
+									<executable>${project.build.directory}/releng-scripts/publish/rsync.sh</executable>
+									<arguments>
+										<arg>-s</arg>
+										<arg>${project.build.directory}/repository</arg>
+										<arg>-t</arg>
+										<arg>mars/snapshots/builds/${JOB_NAME}/${BUILD_ID}-B${BUILD_NUMBER}/all/repo/</arg>
```

Comment:
should this not be a derived property so its not hardcoded to mars/snapshots/builds ?